test/Microsoft.Framework.Configuration.Test/ConfigurationTest.cs

```diff
@@ -12,20 +12,13 @@ namespace Microsoft.Framework.Configuration.Test
     public class ConfigurationTest
     {
         [Fact]
-        public void SetBasePathThroughConstructor()
+        public void SetBasePathThroughBuilder()
```

Comment:
Or otherwise avoid creating that new package and have all the unit tests here instead? Not sure if there is a hard rule about that.

cc @Eilon